### PR TITLE
Update README.md added Examples and mac problems

### PR DESCRIPTION
**Pull request by shadi1989**
This pull request includes a

- [ ] Bug fix
- [x] New feature
- [ ] Translation

The following changes were made

-
-
-

If this is related to an existing ticket, include a link to it as well.

Added a few examples for commandline and Mac Terminal problems

### CODE REVIEW

**Comment by DerScreever** [comment]
If you are already doing this please also write what -dp or -dg do.
But looks good. 
Approved

File: README.md [hunk 1]
```diff
@@ -47,3 +56,10 @@ Note:
 * Can I sign in with Google? Not yet, we're working on it, until then get a Trainer Club account
 
 * I'm on Windows, why does nothing work? See if anything in https://www.reddit.com/r/pokemongodev/comments/4t80df/wip_pokemon_go_map_visualization_google_maps_view/d5feu2f helps
+
+* Problems running on a Mac?
+Try using this in Terminal:
+#Sudo -s
```
Comment (juharris) [inline README.md]: Did you want these to be steps? They currently show as titles.  Use: `1. Sudo -s` ...

**Comment by Octylpy** [comment]
I don't think we should suggest to use `sudo -s`. It should just be `sudo pip install -r requirements.txt`

**Comment by shadi1989** [comment]
@Octylpy Without the `Sudo -s` i cant run it on Mac OS X Sierra(10.12 beta).

**Comment by Octylpy** [comment]
@shadi1989 What happens when you try to run it using just `sudo`? What message do you get?

**Comment by hhu94** [comment]
It is generally recommended to use `--user` when using `pip install`. Installs locally for the current user without the need of `sudo`.

**Comment by JZ6** [comment]
rebase and request another pull please

**Pull request closed without merging**